docs/source/quickstart/index.rst

```diff
@@ -16,46 +16,68 @@ See also the readme in the GitHub repository.
 Install Cloud Custodian
 -----------------------
 
-Cloud Custodian is a Python application and supports Python 3 on Linux, MacOS and
-Windows.
+These instructions will install Cloud Custodian. Cloud Custodian is a Python
+application and supports Python 3 on Linux, MacOS and Windows. We recommend
+using Python 3.6 or higher.
 
-We recommend using Python 3.6 or higher.
+NOTE: Ensure you install the correct follow on package depending on the cloud
+you are deploying to, otherwise you won't have the right modules for that
+specific cloud.
 
 Linux and Mac OS
 +++++++++++++++++++++++++++
 
-To install Cloud Custodian, just run::
+To install Cloud Custodian for AWS, run::
 
-  $ python3 -m venv custodian
-  $ source custodian/bin/activate
-  (custodian) $ pip install c7n       #Install AWS package
-  (custodian) $ pip install c7n_azure #Install Azure package
-  (custodian) $ pip install c7n_gcp   #Install GCP Package
+  python3 -m venv custodian
+  source custodian/bin/activate
+  pip install c7n       # Install AWS package
+
+To install Cloud Custodian for Azure, run::
+
+  python3 -m venv custodian
+  source custodian/bin/activate
+  pip install c7n_azure # Install Azure package
```

Comment:
I'd suggest keeping the `c7n_azure` and `c7n_gcp` installs in the same code block. Otherwise folks may try to create and activate the same environment multiple times and hit problems.

If the aim here is to show that the azure and gcp bits are optional and separate, perhaps a separate line comment in the same code block would be useful? Like...

```
  python3 -m venv custodian
  source custodian/bin/activate
  pip install c7n       # Install AWS package
  
  # Install support for additional providers (optional)
  pip install c7n_azure
  pip install c7n_gcp
```

Reply:
Looks good to me! :sunglasses: